app/com/linkedin/drelephant/tuning/hbt/FitnessManagerHBT.java

```diff
@@ -30,6 +30,7 @@
   private final Logger logger = Logger.getLogger(getClass());
   private boolean isDebugEnabled = logger.isDebugEnabled();
   private final int MINIMUM_HBT_EXECUTION = 3;
+  private final int PARAMETER_RETRY_THRESHOLD = 2;
```

Comment:
Threshold should be little bit higher, as we are already identifying if it is because of auto tuning. May be 3 or 4.